model/translation_request.go

```diff
@@ -25,6 +26,27 @@ type TranslationRequest struct {
 	Team           string
 }
 
+// Validate validates the values of a translation create request.
+func (request *TranslationRequest) Validate() error {
+	if len(request.InstallationID) == 0 {
+		return errors.New("must specify installation ID")
+	}
+	if len(request.Type) == 0 {
+		return errors.New("must specify backup type")
+	}
+	if request.Type == SlackWorkspaceBackupType && len(request.Team) == 0 {
+		return errors.New("must specify team with slack buckup type")
+	}
+	if !strings.HasSuffix(request.Archive, ".zip") {
+		return errors.New("archive must be a valid zip file")
+	}
+	if len(request.Archive) == 4 {
```

Comment:
nit: might be easier to read:
```suggestion
	if request.Archive == ".zip" {
```